### ISSUE TITLE
[Static Site] Icon to denote menu item that will expand to more sub items does not have appropriate or meaningful text

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5806]
## Area or Component

Sub menu (navigation on the left side) across the whole site

## Quick Summary

The short text alternative for the complex image is not appropriate or meaningful.

### How to Replicate
Check any menu item that has third tier menu items and notice that the + icon that is trying to denote that the section will contain more breakdown subsections but because there is missing an alt text this information is not convey to non sighted users.

## Accessibility Violation

The short text alternative for the complex image is not appropriate or meaningful. https://docs.deque.com/issue-help/1.0.0/en/alt-text-short-text-not-meaningful

## Action Requested

Add an aria-label to the icon aria-label="More topics under this category"

## Screenshots (optional)
![Screenshot 2023-05-15 at 12 08 39 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/e17c4e76-1168-4b91-b2ae-332f46dc57ca)

**Comment by johanna-skylight** [comment]
We will probably need @lauraykerr's help to come up with a meaningful alt text

**Comment by lauraykerr** [comment]
@johanna-skylight I've been looking into examples of alt text for this, and read in a couple places that the icon should be hidden because the component should have other characteristics that show whether it's expanded or closed (e.g. `aria-expanded`). 

Is that how our side nav works, too? Or are we relying on the icon to broadcast the state? If it's the latter, can definitely come up with an alt text that takes that into account. Thanks!

**Comment by johanna-skylight** [comment]
@lauraykerr So this one is weird because that menu does not expand, so there is no dynamic behavior happening (ruling out the `aria-expanded`). When the user clicks the button it is taken to another page so the menu item acts as a link. The only pattern that I recognize is that the `+` is displayed when the link points to a page that will contain multiple sections. 

There are two accessibility principles we can follow here:
1- If the image is used as decoration then hide it with the `aria-hidden` true. As you mentioned in the comment.
2- If the image is conveying something for a sighted users then add `alt` text to convey the same to the non-sighted user.

So because of the pattern I mentioned before regarding sub sections it kinda feels it falls on the second principle but I am open to either! I trust your take as well :).

**Comment by lauraykerr** [comment]
@johanna-skylight That makes sense, thanks for clarifying! Just wanted to make sure we're not providing redundant information. I definitely agree that the image conveys meaningful information, and will come up with some alt text imminently!

**Comment by lauraykerr** [comment]
@johanna-skylight What do you think about this alt text: "More topics under this category"?

**Comment by johanna-skylight** [comment]
@lauraykerr  oh I think that sounds perfect!
